scannergradle.properties

```diff
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-gradle/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/SONARGRADL/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-gradle
-archivedVersions=2.8,3.0,3.1,3.1.1,3.2,3.3,3.4.0.2513,3.5.0.2730,4.0.0.2929
-publicVersions=4.2.1.3168
+archivedVersions=2.8,3.0,3.1,3.1.1,3.2,3.3,3.4.0.2513,3.5.0.2730,4.0.0.2929,4.2.1.3168
+publicVersions=4.3.0.3225
+
+4.3.0.3225.description=Support for analysis of Gradle Kotlin DSL files
+4.3.0.3225.date=2023-07-17
+4.3.0.3225.changelogUrl=https://sonarsource.atlassian.net/issues/?jql=project%20%3D%2010137%20AND%20fixVersion%20%3D%2014166
+4.3.0.3225.downloadUrl=https://plugins.gradle.org/plugin/org.sonarqube
```

Comment:
I guess this doesn't link to a binary because that's not how Gradle works(?)

Reply:
yep